lib/main.js

```diff
@@ -200,8 +200,12 @@ export default {
 
   async lint(textEditor, options) {
     const filePath = textEditor.getPath();
-    const text = textEditor.getText();
 
+    if (filePath === null || filePath === undefined) {
```

Comment:
Usually I prefer doing a `value == undefined` since this type-juggles null also. But the linting rules in this project disallow `==` even in the undefined case =[